src/swing-card-component.ts

```diff
@@ -31,4 +31,10 @@ export class SwingCardComponent {
   getCard(): Card {
     return this.swingStack.stack.getCard(this.getNativeElement());
   }
+
+  destroyCard() {
+		this.swingStack.cards.pop();
```

Comment:
It would be called manually like so: this.swingCards[0].destroyCard() but let me check further and confirm back.